pyphare/pyphare/pharesee/hierarchy/hierarchy_compute.py

```
@@ -0,0 +1,97 @@
+        return self.other
+
+
+def _compute_copy_op(patch, op, hinfo, other, reverse=False):
+    def _(a, b):
+        return op(b, a) if reverse else op(a, b)
+
+    data = DataAccessor(hinfo, other)
+    return tuple(
+        {
+            "name": name,
+            "data": _compute_copy_do(pd, lambda ds: _(ds, data[name])),
```

Ruff (0.15.15)

[warning] Function definition does not bind loop variable `name`

(B023)